addons/project/models/project_task.py

```diff
@@ -199,14 +194,14 @@ def _read_group_personal_stage_type_ids(self, stages, domain):
         domain="[('user_id', '=', uid)]", string='Personal Stages', export_string_translation=False)
     # Personal Stage computed from the user
     personal_stage_id = fields.Many2one('project.task.stage.personal', string='Personal Stage State', compute_sudo=False,
-        compute='_compute_personal_stage_id', group_expand='_read_group_personal_stage_type_ids',
+        compute='_compute_personal_stage_id',
+        search='_search_personal_stage_id',
+        group_expand='_read_group_personal_stage_type_ids',
         help="The current user's personal stage.")
-    # This field is actually a related field on personal_stage_id.stage_id
-    # However due to the fact that personal_stage_id is computed, the orm throws out errors
-    # saying the field cannot be searched.
     personal_stage_type_id = fields.Many2one('project.task.type', string='Personal Stage',
-        compute='_compute_personal_stage_type_id', inverse='_inverse_personal_stage_type_id', store=False,
-        search='_search_personal_stage_type_id', default=_default_personal_stage_type_id,
+        related='personal_stage_id.stage_id',
+        search='_search_personal_stage_type_id',
```

Comment:
I think it will not work, that's why we did not convert that computed field into a related one because the `search` method is replaced by `_search_related` one. 😕 🤔 

See https://github.com/odoo/odoo/blob/master/odoo/orm/fields.py#L620-L620

Reply:
Will fix it in the ORM to keep the overrides defined on the field.

Reply:
Cool, thanks 🙂

Reply:
We shouldn't accept specific search method for related field. That breaks the semantic of the related field.